### PR TITLE
Nopatch kernel to address CVE-2022-3110, CVE-2022-3707

### PR DESCRIPTION
**Pull request by Camelron**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

What does the PR accomplish, why was it needed?

###### Change Log  

- Address CVE-2022-3110, CVE-2022-3707

###### Does this affect the toolchain?  

NO

###### Links to CVEs  
- https://nvd.nist.gov/vuln/detail/CVE-2022-3110
- https://nvd.nist.gov/vuln/detail/CVE-2022-3707

### CODE REVIEW

File: SPECS/kernel/CVE-2022-3707.nopatch [hunk 1]
```diff
@@ -0,0 +1,3 @@
+CVE-2022-3108 - Fix present in 5.10.174
```
Comment (rlmenge) [inline SPECS/kernel/CVE-2022-3707.nopatch:1]: copy paste typo

File: SPECS/kernel/CVE-2022-3108.nopatch [hunk 2]
```diff
@@ -0,0 +1,3 @@
+CVE-2022-3108 - Fix present in 5.10.174
+Upstream: 8027a9ad9b3568c5eb49c968ad6c97f279d76730
```
Comment (rlmenge) [inline SPECS/kernel/CVE-2022-3108.nopatch:2]: This does not match the patch posted on NIST and looks to be for a different CVE

https://nvd.nist.gov/vuln/detail/CVE-2022-3108

**Review by rlmenge — CHANGES_REQUESTED** [review]

File: SPECS/kernel/CVE-2022-3108.nopatch (hunk 2 above)
Comment (Camelron) [reply SPECS/kernel/CVE-2022-3108.nopatch:2]: uh oh, will retrace my steps.

Comment (Camelron) [reply SPECS/kernel/CVE-2022-3108.nopatch:2]: Got it mixed up with a very similar commit: https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/commit/?h=v5.10.176&id=25bc9fd4c8d13036c0656a00ebedc2cff4650a91

**Review by rlmenge — APPROVED** [review]

**Pull request merged**